src/components/TransactionItem/Details/Details.js

```diff
@@ -32,6 +32,8 @@ type Props = {|
   +isSent: boolean,
   +isActive: boolean,
   +isFromFavorites: boolean,
+  +isEventMint: boolean,
+  +isEventBurn: boolean,
 |}
```

Comment:
don't need to pass both isEventMint/Burn
it's possible to check `data.eventType === 2` that means `isMintable`